plugins/woocommerce/src/Admin/Features/Blueprint/SettingOptions.php

```diff
@@ -4,162 +4,89 @@
 
 namespace Automattic\WooCommerce\Admin\Features\Blueprint;
 
-use Automattic\WooCommerce\Blueprint\UseWPFunctions;
-use Automattic\WooCommerce\Blueprint\Util;
-use WC_Admin_Settings;
-use WC_Settings_Page;
-
 /**
  * Handles getting options from WooCommerce settings pages.
  *
  * Class SettingOptions
  */
 class SettingOptions {
-	use UseWPFunctions;
+	/**
+	 * Setting option controller.
+	 *
+	 * @var \WC_REST_Setting_Options_Controller
+	 */
+	private $setting_option_controller;
 
 	/**
-	 * Array of WC_Settings_Page objects.
+	 * Ignore setting types.
 	 *
-	 * @var WC_Settings_Page[]
+	 * @var array
 	 */
-	private array $setting_pages;
+	private $ignore_setting_types = array( 'title', 'sectionend', 'slotfill_placeholder' );
```

Comment:
I don't think this is relevant anymore right given we make use of the settings controller now.
It looks like any of the above items aren't in the settings type list: https://github.com/woocommerce/woocommerce/blob/0a9c0049b300f2f42dd74899485e2eb1278a5708/plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-setting-options-controller.php#L236

Reply:
The API endpoint's list of valid setting types appears to be incomplete, as it omits several supported type:

- email_image_url
- email_font_family  
- relative_date_selector

 (We probably need to update the API endpoint.)

Also, we have to ignore types here because we use `get_group_settings` instead of the API endpoint (get_items). The `get_group_settings` method returns all settings without filtering out the invalid types.

We need to use `get_group_settings` because the API endpoint does not include the `option_key` required to retrieve database values.